packages/stream/create-stream/create-stream.js

```diff
@@ -0,0 +1,114 @@
+import { Internal } from '../../internal/index.js'
+import { InitializationInstance } from '../../app/init/init.js'
+
+/**
+ * Function to cek if the input text has a space or not. If yes, then will replace space with "-"
+ *
+ * @function
+ * @param {string} text -- stream's name
+ * @returns {string} -- stream's slug as per stream's name but without space
+ */
+function replaceWhiteSpace(text) {
+  return /\s/.test(text) ? text.split(' ').join('-') : text
+}
+
+/**
+ * @typedef Config
+ * @property {string} name -- name of stream
+ * @property {string} slug -- slug of stream
+ * @property {string} description -- description of stream
+ */
+
+/**
+ * @typedef FetchResponse
+ * @property {object} status -- A status response
+ * @property {object} data -- A return data as per endpoint
+ */
+
+/**
+ * A create stream module that should pass stream's name parameter, and user could pass the stream's slug & description parameter as well
```

Comment:
Actually the description doesn't need to be long and specific (doesn't need to mention about the user as well). Just describe this as a function to create a new stream. Something simple like that.

Reply:
@faiq-naufal okay mas, done at commit [79736af](https://github.com/inlivedev/inlive-js-sdk/pull/11/commits/79736afa6c7dc79e6a50e7bfa93384b128f7e5d2)